Week-2/exercise-2.2/README.md

```diff
@@ -0,0 +1,3 @@
+square boxes animation
+-while hover on the box-container the boxes will rotate to left and right
+-i used transitions and transform
```

Comment:
Like the first application , you should have hosted in the netlify so that it eases my efforts to review.